pa11y/webapp/write-report.js

```diff
@@ -22,7 +22,6 @@ const urls = [
   'https://wellcomecollection.org/events/Wqkd1yUAAB8sW4By',
   'https://wellcomecollection.org/event-series/WlYT_SQAACcAWccj',
   'https://wellcomecollection.org/concepts/n4fvtc49',
-  'https://wellcomecollection.org/works/a2239muq/items',
```

Comment:
We're already testing `https://wellcomecollection.org/works/e7vav3ss/items?page=1&canvas=1&sierraId=b28136615&langCode=eng`. Is that what you meant for adding an item viewer page @alexwlchan ? I might've gotten it wrong.

Reply:
Yup, that looks fine, although I have no memory of suggesting that. 😅